search-app/src/cmr/search/services/parameters/converters/geometry.clj

```diff
@@ -35,16 +38,30 @@
   [^LineString line-string]
   (rr/ords->ring :geodetic (coords->ords (.getCoordinates line-string))))
 
+(defn force-ccw-orientation
+  "Forces a LineString to be in counter-clockwise orientation"
+  [^LineString line-string winding]
+  (let [coords (.getCoordinates line-string)]
+    (if (and 
+          (> (count coords) 3)
+          (= winding :cw))
+      (.reverse line-string)
+      line-string)))
+
 (defn polygon->shape
-  "Convert a JTS Polygon to a spatial lib shape that can be used in a Spatial query"
-  [^Polygon polygon]
-  (let [boundary-ring (.reverse (.getExteriorRing polygon))
+  "Convert a JTS Polygon to a spatial lib shape that can be used in a Spatial query.
+  The `context` map can be used to provide information about winding. Accepted keys
+  are `:boundary-winding` and `:hole-winding`. Accepted values are `:cw` and `ccw`."
+  [^Polygon polygon context]
```

Comment:
It's been awhile, but I thought the `context` parameter had a specific meaning in the CMR (a certain type of map with the full system context for a request). If that's not what context is being used for in this case, I'd suggest renaming it to `options` (assuming it's a map that allows for different behavior within the functions as implied by a later docstring).

Reply:
Done